supergloo/docs/mesh/install.md

```diff
@@ -0,0 +1,266 @@
+---
+title: "Installing a Mesh"
+weight: 2
+---
+
+# Overview
+
+SuperGloo can be used to install, upgrade, and uninstall a supported mesh.
+
+Currently supported meshes for installation:
+
+- Istio
```

Comment:
A list with one entry looks a bit weird, maybe we can add other elements with "WIP" or "planned" labels. Even better, we could reference [the feature table in the main README](https://github.com/solo-io/supergloo#features-snapshot) (which we should update!).